home_work/hw_23/tests/hw_24/test_create_contact_001.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from loguru import logger
+
+from home_work.hw_23.pages.add_user_page import AddUserPageHelper
+from home_work.hw_23.pages.common_el_page import CommonElementsHelper
+from home_work.hw_23.pages.contact_add_page import AddContactHelper
+from home_work.hw_23.pages.contact_list_page import ContactListHelper
+from home_work.hw_23.urls.site_page_urls import PageUrls
+from home_work.hw_23.utils.constants.site_headers_names import Headers
+
+
+@pytest.mark.parametrize("browsers_chrome", [1], indirect=True)
```

Comment:
in the current version of the solution, it makes little sense and is not necessary, now it looks like an overcomplication